### PR TITLE
feat: Update Jetty to Jetty 11 and ktor to ktor-2.0.2-jitsi

### PR DESCRIPTION
**Pull request by JonathanLennox**

### CODE REVIEW

**Review by damencho — DISMISSED** [review]

**Comment by JonathanLennox** [comment]
Note: I've asked @oanaianc to test whether the webhooks still work after this change - I'm concerned they won't, because the ktor-server-jetty package references jetty 9.x.  Please don't merge this until that test is complete.

**Comment by JonathanLennox** [comment]
Note: this now needs the ktor from my PR at https://github.com/ktorio/ktor/pull/3015.

**Comment by JonathanLennox** [comment]
Note this is now on top of https://github.com/jitsi/jibri/pull/483.

File: src/main/kotlin/org/jitsi/jibri/CallUrlInfo.kt [hunk 1]
```diff
@@ -53,6 +54,6 @@ data class CallUrlInfo(
 
     override fun hashCode(): Int {
         // Purposefully ignore urlParams here
-        return Objects.hash(baseUrl.toLowerCase(), callName.toLowerCase())
+        return Objects.hash(baseUrl.lowercase(Locale.getDefault()), callName.lowercase(Locale.getDefault()))
```
Comment (bbaldino) [inline src/main/kotlin/org/jitsi/jibri/CallUrlInfo.kt:57]: did you explicitly pass the locale in for a reason? i see it supports a plain version without passing the locale as well.

**Review by bbaldino — DISMISSED** [review]
lgtm. wish i remembered what jersey was used for...part of me wonders if that dep might be obsolete.

File: src/main/kotlin/org/jitsi/jibri/CallUrlInfo.kt (hunk 1 above)
Comment (JonathanLennox) [reply src/main/kotlin/org/jitsi/jibri/CallUrlInfo.kt:57]: That's what IntelliJ's auto-fix of the deprecation warning did -- I can change it.

**Review by aaronkvanmeerten — APPROVED** [review]

**Pull request merged**